### ISSUE TITLE
Svelte client-side fetching not working in v6

### ISSUE THREAD

**Issue by samlfair** [prismicio/prismic-javascript#180]
In my SvelteKit project, I can get the native `fetch` to work, but when I pass `fetch` to the Client, I get this error in the console:

```
Uncaught (in promise) TypeError: Failed to execute 'fetch' on 'Window': Illegal invocation
    at Client.fetch (index.es.js:497)
    at Client.getRepository (index.es.js:323)
    at Client.getCachedRepository (index.es.js:432)
    at Client.getResolvedRefString (index.es.js:485)
    at Client.buildQueryURL (index.es.js:364)
    at Client.get (index.es.js:265)
    at Client.getAll (index.es.js:279)
    at getData (clientside.svelte:13)
```

This is in the clientside route in this repo:

https://github.com/samlfair/svelte-docs-tester-01

**Comment by angeloashmore** [comment]
Thanks @samlfair, this is fixed in `v6.0.0-alpha.3` (via https://github.com/prismicio/prismic-javascript/commit/83c6290ec79a4f5bbecc1d2d9308816300101138).

**Comment by angeloashmore** [comment]
Oops, spoke too soon! I forgot I edited a bit of your project while debugging.

It's actually fixed in `v6.0.0-alpha.4` (via https://github.com/prismicio/prismic-javascript/commit/d22ae21483f967b4366d501f7e98934d6ee668bb) with no changes to your project except updating the `@prismicio/client` version.
